snuba/cli/config.py

```diff
@@ -55,18 +57,33 @@ def get(*, key: str, format: str) -> None:
 @config.command()
 @click.argument("key")
 @click.argument("value")
-def set(*, key: str, value: str) -> None:
+@click.option(
+    "--force", is_flag=True, default=False, help="Override type checking of values"
+)
+def set(*, key: str, value: str, force: bool) -> None:
     "Set a single key."
-
-    state.set_config(key, value, user=get_user())
+    try:
+        typed_value = numeric(value)
+        state.set_config(key, typed_value, user=get_user(), force=force)
```

Comment:
Do you need to call `numeric(value)` and pass that to set_config here? Why not have `state.set_config()` handle everything?

Reply:
Moved the conversion to typed value in state.set_config